de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolRequest/presenter/ToolRequestPresenterImpl.java

```diff
@@ -122,12 +121,7 @@ public void setViewDebugId(String baseId) {
 
     @Override
     public void onAdminMakeToolPublicSelected(AdminMakeToolPublicSelectedEvent event) {
-        adminView = adminFactory.create(new ListStore<>(new ModelKeyProvider<Tool>() {
-            @Override
-            public String getKey(Tool item) {
-                return item.getId();
-            }
-        }));
+        adminView = adminFactory.create(new ListStore<>(item -> item.getId()));
```

Comment:
This syntax (works only for interfaces with one method only. So it knows which method to override. See https://docs.oracle.com/javase/tutorial/java/javaOO/lambdaexpressions.html#approach6
also See http://docs.sencha.com/gxt/4.x/javadoc/gxt-4.0.2/index.html